src/components/Layout/Footer/index.tsx

```diff
@@ -44,10 +44,10 @@ const Footer: React.FC<FooterProps> = ({ scrollTop }) => {
             <Grid container>
               <Grid item md={12} xs={12}>
                 <Box className={classes.floatBox}>
-                  <Typography variant='h3'>
+                  {/* <Typography variant='h3'>
```

Comment:
why do we want to keep the code comments?

Reply:
Let's remove them 👌🏻